### PR TITLE
Removed References to Riemann Port in Manager Blueprint References

### PR DESCRIPTION
**Pull request by EarthmanT**

### CODE REVIEW

File: source/guide/3.2/reference-openstack-manager.md [hunk 1]
```diff
@@ -115,4 +114,4 @@ The "Openstack manager" blueprint contains the following nodes:
 The *manager's* node *configure* lifecycle operation is mapped to a method in the configure.py module which takes the following actions:
 
   - It sets the *provider context*, which will be used by the Openstack plugin when installing applications at later stages.
-  - It creates a file on the Cloudify manager server, which holds the configuration settings for connecting with Openstack. This file will be used by the Openstack plugin when installing applications at later stages.
```
Comment (iliapolo) [inline source/guide/3.2/reference-openstack-manager.md]: why was this line removed?

**Comment by iliapolo** [comment]
looks like there are conflicts...

**Comment by nir0s** [comment]
Also, I think we need to verify that our manager blueprint don't actually open port 5555 :)

**Comment by EarthmanT** [comment]
https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/60
:)

**Comment by EarthmanT** [comment]
fixed conflicts

**Comment by nir0s** [comment]
there are conflicts here as well. cloudify-cosmo/cloudify-manager-blueprints#60
